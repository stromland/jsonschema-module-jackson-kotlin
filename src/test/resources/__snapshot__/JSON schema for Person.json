{
  "$schema" : "https://json-schema.org/draft/2020-12/schema",
  "type" : "object",
  "properties" : {
    "age" : {
      "type" : "integer",
      "default" : 28
    },
    "gender" : {
      "type" : [ "string", "null" ]
    },
    "lastName" : {
      "type" : "string"
    },
    "name" : {
      "type" : "string"
    }
  },
  "required" : [ "lastName", "name" ]
}