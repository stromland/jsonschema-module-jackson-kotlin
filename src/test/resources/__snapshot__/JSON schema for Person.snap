{
  "$schema" : "http://json-schema.org/draft-07/schema#",
  "type" : "object",
  "properties" : {
    "age" : {
      "type" : "integer",
      "default" : 28
    },
    "lastName" : {
      "type" : "string"
    },
    "name" : {
      "type" : "string"
    },
    "sex" : {
      "type" : [ "string", "null" ]
    }
  },
  "required" : [ "lastName", "name" ]
}